src/Common/perf/PerfRunner/project.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.0.0-*",
```

Comment:
In order to reference it you will need to add the new package dependency at https://github.com/dotnet/corefx/blob/master/external/test-runtime/project.json. Once that project is restored you should be able to find the reference.